README.md

```diff
@@ -90,7 +90,7 @@ List of options provided by CMake:
 | - | - | - | - |
 | UMF_BUILD_SHARED_LIBRARY | Build UMF as shared library | ON/OFF | OFF |
 | UMF_BUILD_OS_MEMORY_PROVIDER | Build OS memory provider | ON/OFF | ON |
-| UMF_BUILD_LIBUMF_PROVIDER_LEVEL_ZERO | Build Level Zero memory provider | ON/OFF | OFF |
+| UMF_BUILD_LEVEL_ZERO_PROVIDER | Build Level Zero memory provider | ON/OFF | OFF |
```

Comment:
Please do not mix this PR with #291. I have split it into 2 PRs on purpose to be able to fix ONLY the name.

Reply:
heh, this PR fixes the README and it contains the improper info now...? 😉

Reply:
If you change something around (in this case `OFF` to `ON`) it will conflict with future change. I agree with @ldorau

Reply:
I really do NOT have to fix all existing issues in one PR.
This PR has name "Fix name of UMF_BUILD_LEVEL_ZERO_PROVIDER in the README.md file" and it will fix only this name and **nothing more**.

Reply:
> heh, this PR fixes the README and it contains the improper info now...? 😉

This PR does not have the name "Fix all issues in README.md file"